test/e2e/configure_cni_test.go

```diff
@@ -86,6 +91,22 @@ func testConfigureCNI(t *testing.T) {
 	})
 }
 
+// testKubeletUninstall tests if WMCB returns an error if the kubelet is uninstalled
+func testKubeletUninstall(t *testing.T) {
+	t.Run("Kubelet service uninstalls", func(t *testing.T) {
+		svcMgr, err := mgr.Connect()
+		require.NoError(t, err)
+		defer svcMgr.Disconnect()
+		kubeletSvc, err := bootstrapper.AssignExistingKubelet(svcMgr)
+		require.NoError(t, err)
+		require.NotNil(t, kubeletSvc)
+
+		err = kubeletSvc.UninstallKubeletService()
+		assert.NoError(t, err, "unable to uninstall kubelet service")
+		assert.False(t, svcExists(t, bootstrapper.KubeletServiceName))
```

Comment:
Here too you should be using the pattern:
```go
bootstrapper.NewWinNodeBootstrapper()
wmcb.UninstallKubelet()
wmcb.Disconnect()
```
This is how someone using WMCB as a library would use it.